components/dashboard/src/admin/UserSearch.tsx

```diff
@@ -46,9 +52,9 @@ export default function UserSearch() {
         try {
             const result = await getGitpodService().server.adminGetUsers({
                 searchTerm,
-                limit: 50,
+                limit: pageLength,
                 orderBy: "creationDate",
-                offset: 0,
+                offset: (currentPage - 1) * pageLength,
```

Comment:
`currentPage` starts at value of 0, wouldn't this cause the offset to be negative?

Reply:
currentPage should start at 1.

Reply:
Now it does, but before it was definitely initialized to 0.

Reply:
Yes on 3 out of 4 occasions 😬 (the four search pages are a bit copy and paste and of course I only tested on one of them, which also explains why the limit was set to `2` only on one of them)